providers/install.rb

```diff
@@ -20,12 +20,14 @@
 action :run do
   # Package install
   if node['redisio']['package_install']
-    package "redisio_package_name" do
+    package_resource = package 'redisio_package_name' do
```

Comment:
Should this be `action :nothing` with a `.run_action(:install)` though? You're not actually running the action here, so it won't be updated unless you run the action, vs. just putting it on the resource collection (what the DSL does).